tests/copyright/policy

```diff
@@ -67,9 +67,12 @@ tests/collect-cov.sh
 tests/copyright/check.sh
 modules/java/(tools|[^/]*$)
 modules/java-modules/(dummy|elastic|elastic-v2|hdfs|http|kafka|[^/]*$)
-modules/(afamqp|affile|afmongodb|afprog|afsmtp|afsocket|afsql|afstomp|afstreams|afuser|basicfuncs|cef|confgen|cryptofuncs|csvparser|date|diskq|dbparser|geoip|geoip2|graphite|json|kvformat|linux-kmsg-format|pacctformat|pseudofile|python|redis|riemann|syslogformat|systemd-journal|getent|system-source|stardate|snmptrapd-parser|xml|openbsd|[^/]*$)
+modules/(afamqp|affile|afmongodb|afprog|afsmtp|afsocket|afsql|afstomp|afstreams|afuser|basicfuncs|cef|confgen|cryptofuncs|csvparser|date|diskq|dbparser|geoip|geoip2|graphite|json|kvformat|linux-kmsg-format|pacctformat|pseudofile|redis|riemann|syslogformat|systemd-journal|getent|system-source|stardate|snmptrapd-parser|xml|openbsd|[^/]*$)
 modules/(add-contextual-data|tagsparser|map-value-pairs|hook-commands|appmodel|[^/]*$)
+modules/python/python-(config|debugger|dest|global-code-loader|helpers|logmsg|logparser|main|module|parser|plugin|tf|value-pairs|grammar)[^/]*$
+modules/python/(pylib|compat|tests|sngexample.py)
 scl
 scripts
  GPLv2+_SSL,non-balabit
 modules/http
+modules/python/python-stats.*$
```

Comment:
Is there any reason to exclude 
```
* Copyright (c) 2018 Balabit
```
from the copyright headers? You can still have a line with your name in it, together with this line, and then we could avoid this complex change in the policy checker.